migrate_formspecs_lbm.lua

```diff
@@ -0,0 +1,14 @@
+
+minetest.register_lbm({
+    label = "Migrate travelnet formspecs from meta to rightclick/punch-only",
+    name = "travelnet:migrate_formspecs",
+    nodenames = {"group:travelnet"},
+    action = function(pos)
+        -- check formspec meta-field and clear it
+        local meta = minetest.get_meta(pos)
+        local legacy_formspec = meta:get_string("formspec")
+        if not travelnet.is_falsey_string(legacy_formspec) then
+            meta:set_string("formspec", "")
+        end
```

Comment:
Is there any reason to check the string? Otherwise this would be better:
```suggestion
        -- clear formspec meta-field
        minetest.get_meta(pos):set_string("formspec", "")
```

Reply:
not sure about that, i just copied the existing code-snippet, clearing the field only if it is set sounds sane IMO

Reply:
Depends on how engine handles useless updates, if it marks mapblock as changed because of this then it is better to check. Might be intelligent enough to not do that if data is not really changing, simpler checking however would be good even if check stays:
```suggestion
        -- check formspec meta-field and clear it
        local meta = minetest.get_meta(pos)
        local legacy_formspec = meta:get("formspec")
        if legacy_formspec then
            meta:set_string("formspec", "")
        end
```

Reply:
i'm neutral to both approaches, i used the simpler one from @OgelGames :+1: